Pre-compilation for Windows error

**I'm submitting a ...**
```
[X] bug report
[ ] feature request
[ ] support request
```

**Current behavior**
Following the advice on how to [speed up the build on Windows](https://github.com/mgechev/angular-seed/wiki/Speed-up-the-build-on-Windows) it is adviced to run `tsc --project tsconfig.json`. On my machine this returns the following error:

```
41     findTestabilityInTree(elem: Node, findInAncestors?: boolean): Testability;
                                   ~~~~

node_modules/@angular/core/src/testability/testability.d.ts(41,33): error TS2304: Cannot find name 'Node'.
```

**Minimal reproduction of the problem with instructions**

* git clone --depth 1 https://github.com/mgechev/angular-seed.git
* cd angular-seed
* npm install
* tsc --project tsconfig.json

**Please tell us about your environment:**

* **Angular Seed Version:** `18a6e44da97d2734d7e81377df49e52ac70d2354`

* **Node:** `node --version` = v7.1.0

* **Typescript:** `tsc -v` = Version 2.1.5

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -24,6 +24,20 @@
       "node"
     ]
   },
+  "files": [
+    "gulpfile.ts",
+    "tools/manual_typings/seed/autoprefixer.d.ts",
+    "tools/manual_typings/seed/cssnano.d.ts",
+    "tools/manual_typings/seed/express-history-api-fallback.d.ts",
+    "tools/manual_typings/seed/istream.d.ts",
+    "tools/manual_typings/seed/karma.d.ts",
+    "tools/manual_typings/seed/merge-stream.d.ts",
+    "tools/manual_typings/seed/open.d.ts",
+    "tools/manual_typings/seed/operators.d.ts",
+    "tools/manual_typings/seed/slash.d.ts",
+    "tools/manual_typings/seed/systemjs-builder.d.ts",
+    "tools/manual_typings/seed/tildify.d.ts"
+  ],
   "exclude": [
     "node_modules",
     "dist",

